libs/sdk-ui-tests/scenarios/pivotTable/autoresize.tsx

```diff
@@ -22,7 +22,7 @@ export default scenariosFor<IPivotTableProps>("PivotTable", PivotTable)
         ...PivotTableWithSingleColumn,
         config: {
             columnSizing: {
-                defaultWidth: "viewport",
+                defaultWidth: "autoresizeAll",
```

Comment:
Do we have tests for `viewport` option?

Reply:
yes, here https://github.com/gooddata/gooddata-ui-sdk/pull/1455/files#diff-5586818ecff5be915a401c34cf1830a898d621c473c142845de6b665c8ab0928R116